.github/workflows/release.yml

```diff
@@ -21,6 +21,9 @@ jobs:
           persist-credentials: false
           fetch-depth: 0
 
+      - name: Create zip archive
```

Comment:
I think it is a good idea to keep the release process is identical across all our modules.